modules/bgsegm/tutorials/bgsegm_bg_subtraction/bgsegm_bg_subtraction.markdown

```diff
@@ -31,7 +31,7 @@ In the following, we will have a look at two algorithms from the `bgsegm` module
 
 It is a Gaussian Mixture-based Background/Foreground Segmentation Algorithm. It was introduced in
 the paper "An improved adaptive background mixture model for real-time tracking with shadow
```

Comment:
the paper "An Improved Adaptive Background Mixture Model for Realtime Tracking with Shadow
Detection" by P. KaewTraKulPong and R. Bowden in 2002. It uses a method to model each background